src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -249,6 +249,15 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
   const existingPostRun = moduleConfig.postRun || [];
   (moduleConfig as any).preloadPlugins = [];
 
+  let resourcesLoaded = 0;
+  function setProgress(){
+      resourcesLoaded++;
+      const percentage = resourcesLoaded / totalResources.length * 100;
+      document.documentElement.style.setProperty('--blazor-load-percentage', `${percentage}%`);
+      document.documentElement.style.setProperty('--blazor-load-percentage-text', `${Math.round(percentage)}%`);
+      document.documentElement.style.setProperty('--blazor-load-percentage-color', '#1b6ec2');
```

Comment:
When the progress bar first launches, the entire circle is outlined in blue, instead of showing the gray outline that denotes the initial state. The blue outline indicating progress becomes accurate once `--blazor-load-percentage` is set in `setProgress`. But before that CSS variable is set, it looks like the page is 100% loaded as `stroke-dasharray` isn't modified. I set the stroke color after the percentage is calculated in order to avoid this problem. Another solution I had previously was to initially set the blue circle to `display: none`, and change it to `display: block` in setProgress, but that didn't seem to work anymore without `getElementById`.

I can add set the color value in the template CSS if it is okay to have the blue outline displayed before the correct percentage is set.